modules/post/windows/recon/host_info.rb

```diff
@@ -0,0 +1,149 @@
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::System
+  include Msf::Post::Windows::Powershell
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Get the sessions current user',
```

Comment:
Copying what the Linux module does:
```suggestion
        'Name' => 'Update information about session host',
```